### DISCUSSION TITLE
Section numbers not generated automatically

### DISCUSSION THREAD

**Discussion by etiennethomassen** [Zettlr/Zettlr#3604] [Q&A] [answered]
Zettlr seems reallly interesting and I am trying to set it up for both short notes and reports. I am having problems with the report side of things. I am wondering if there are examples available for a report with title page, toc, numbered headings and citations.

For now I am having trouble with headings. They do not receive a number automatically:
![afbeelding](https://user-images.githubusercontent.com/1319600/178573321-1011afd1-05a1-4dee-a246-f7e0d0b0bd64.png)

What am I missing?

I am on Ubuntu 22.04 (installed from .deb)

**Comment by jgsw** [comment]
I think this is a bug.  I can confirm that in Zettlr 2.3.0, the variable number-sections does not have any effect for me either, when used in the YAML block in a document. 

As a workaround, you can alter the defaults in the Assets Manager (>File>Preferences>Assets Manager). Make sure that number-sections: true  is set in the defaults file for PDF. This will then output all PDFs with numbered sections.

**Comment by nathanlesage** [comment, marked as answer]
`numbered-sections` is a property that can only be defined in a defaults file, but not in the YAML frontmatter. The reason is that the property does not control anything related to the actual content. Rather, it affects the template as a whole, and as such it needs to be specified in the defaults file. In this regard, Pandoc is always a tad difficult to read, I agree.

A good rule of thumb: Anything that modifies the layout/style/external things of a file needs to go into a defaults file, anything that comprises information about or for the document in question goes into the YAML frontmatter. (This includes any template variables, unfortunately, so the distinction is very fuzzy.)

**Reply by etiennethomassen** [reply]
Thank you! Now it works :-)

For me I think it would be more logical to control this setting on a project/document level in stead of a general defaults file. Can that be done somehow?

**Comment by DutchPete** [comment]
I always have the headings/subheadings numbered in the body of  my files. Is there a way to have Zettlr do the same, but independent of export? In other words, I would like to see numbered headings on my screen.

After all, the Zettlr Table of Contents in the outline pane numbers all the headings.
